index.html

```diff
@@ -9,21 +9,21 @@
     <title>Mourid Portfolio</title>
 </head>
 <body>
-        <header class="navbar">
+        <nav class="navbar">
         <div class="container">
             <div class="logo"><a href="#"><img src="img/My Logo.png"></a></div>
         <div class="menu"><img src="img/Union.svg" alt=""></div>
     </div>
-</header>
+</nav>
     <header class="header">
```

Comment:
*[OPTIONAL]*

- Please, kidly rename this header to a div as you've used it to divide your page, that's not the ideal way of using the header tag. the div tag is more semantically correct when you're just using it as a divider in a different part of your HTML structure. Please, let's fix this to improve our semantic usage.